shell/src/main/java/alluxio/cli/fsadmin/report/MetricsCommand.java

```diff
@@ -74,8 +74,7 @@ public int run() throws IOException {
           // Bytes long can be transformed to human-readable format
           strValue = FormatUtils.getSizeFromBytes((long) doubleValue);
           if (name.contains(THROUGHPUT_METRIC_IDENTIFIER)) {
-            // throughput is calculated as one-minute exponentially-weighted moving average rate
-            strValue = strValue + "/min";
+            strValue = strValue + "/SEC";
```

Comment:
nit: is there a reason to make this all caps?

Reply:
Because `FormatUtils.getSizeFromBytes()` will return size in caps like `B, MB, GB`

Reply:
also, is every gauge at the `sec` granularity? Is that something we even want?

I feel like there will be times when a cluster is not under a high load and many of the gauges will report as 0 - especially for people who scrape metrics on a periodic basis, second-level granularity seems like it would cause lots of information to be missed.

Reply:
All the cluster throughput is at `sec` now. Throughput can be of sec or minute. Yeah, maybe use `MIN` make more sense since alluxio is usually long-running cluster.